src/Invoice.js

```diff
@@ -0,0 +1,165 @@
+            <div className="container-invoice-data-top">
+                <div className="container-invoice-data-top-inner">
+                    <p className="text color-blue"><span>#</span><span className="bold dark">{invoiceId}</span></p>
+                    <span className="text color-blue">{invoice.description}</span>
+                </div>
+                <div className="address">
+                    <span className="text-information">{invoice.senderAddress.street}</span>
+                    <span className="text-information">{invoice.senderAddress.city}</span>
+                    <span className="text-information">{invoice.senderAddress.postCode}</span>
+                    <span className="text-information">{invoice.senderAddress.country}</span>
+                </div>
+            </div>
+            <div className="date">
+                <p className="text color-blue">Invoice Date</p>
+                <p className="text-date pt-10 pb-22">{invoice.createdAt}</p>
+                <p className="text color-blue">Payment Due</p>
+                <p className="text-date pt-10">{invoice.paymentDue}</p>
+            </div>
+            <div className="user-data">
+                <span className="text color-blue">Bill To</span>
+                <span className="text-date pt-10 pb-8">{invoice.clientName}</span>
+                <span className="text-information">{invoice.clientAddress.street}</span>
+                <span className="text-information">{invoice.clientAddress.city}</span>
+                <span className="text-information">{invoice.clientAddress.postCode}</span>
+                <span className="text-information">{invoice.clientAddress.country}</span>
+            </div>
+            <div className="email">
+                <span className="text color-blue">Sent to</span>
+                <span className="text-date pt-10">{invoice.clientEmail}</span>
+            </div>
+            <div className="payment-cart">
+                <div className="mobile">
+                    {invoice.items.map(item => (
+                        <>
+                        <div className="payment-cart-inner">
+                            <div className="payment-cart-info">
+                                <p className="text bold dark pb-8">{item.name}</p>
+                                <p className="text bold color-blue">{item.quantity} x £ {item.price}</p>
+                            </div>
+                            <p className="text bold dark position-right pt-10">£ {item.total}</p>
+                            </div>
+                        </>
+                    ))}   
+                </div>
+                <div className="tablet">
+                    <div className="payment-cart-information">
+                        <p className="text color-blue pb-32">Item Name</p>
+                        {invoice.items.map(item => (
+                            <p className="text bold dark pb-32">{item.name}</p>
```

Comment:
React trazi 'key' property tamo gde se elementi renderuju u petlji
```suggestion
                            <p key={item.id} className="text bold dark pb-32">{item.name}</p>
```